src/bindings/overlay.js

```diff
@@ -63,9 +68,14 @@ export default function overlayElement(element, translation) {
     }
   }
 
-  for (const key in translation.attrs) {
-    if (isAttrAllowed({ name: key }, element)) {
-      element.setAttribute(key, translation.attrs[key]);
+  if (translation.attrs === null) {
+    return;
+  }
+
+  for (const [ns, name, value] of translation.attrs) {
```

Comment:
I've had bad experiences using destructing assignment in hot code before.  You probably tested this and still went for this syntax here -- does this mean the performance is better nowadays?